runtime/services/catalog/migrator/sources/sources_test.go

```diff
@@ -88,6 +88,9 @@ func TestConnectorWithSourceVariations(t *testing.T) {
 		//{"duckdb", "", map[string]any{
 		//	"sql": `select * from read_csv_auto('gs://scratch.rilldata.com/rill-developer/AdBids.csv.gz')`,
 		//}},
+		//{"duckdb", "", map[string]any{
```

Comment:
Why this comment ?

Reply:
Just added to test if this works or not, commented as Suggested by Aditya as it only when we want to test locally its working for such url or not)